src/parser.ts

```diff
@@ -32,7 +32,7 @@ interface PrintNode {
   schema: Field,
 }
 
-class SubstraitParser {
+export class SubstraitParser {
```

Comment:
In typescript we don't use `module.exports`.  Instead, we can use the `export` keyword to mark which things should be exported.